environment.yml

```diff
@@ -0,0 +1,29 @@
+# Conda environment for SPC
+name: SPC
+channels:
+ - conda-forge
+dependencies:
+ # Environment management tools
+ - python=3.8
+ - pip=22.0.3
+ # Jupyter & interactive stuff
+ - jupyterlab=3.2.9
```

Comment:
There's a bunch of random dependencies in here that have nothing to do with using SPC. I don't think it's our job to get people started with conda without having any idea what they're trying to do

Reply:
Again, it is optional, this might be use for researchers where conda is the most common environment for python.